roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```diff
@@ -2,12 +2,12 @@
   ansible.builtin.include_role:
     name: common
   vars:
-    common_cr_test_id: RHOSO-xxxx
+    common_cr_test_id: ""
```

Comment:
If https://github.com/infrawatch/feature-verification-tests/pull/255 gets merged first, you can remove this line and it should pick up the default.

Use a Depends-On to test this suggestion